packages/hw-transport-node-hid/package.json

```diff
@@ -26,7 +26,9 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@ledgerhq/hw-transport": "^4.19.0",
-    "node-hid": "^0.7.2"
+    "lodash.debounce": "^4.0.8",
```

Comment:
it's a lil detail, but I would prefer we depend on `lodash` and doing `import debounce from "lodash/debounce"` , like on live